app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = [Person(p["name"], p["age"]) for p in people]
```

Comment:
The list comprehension correctly creates `Person` objects, but the function does not return this list correctly. Ensure that the `persons` list is returned at the end of the function.